src/test/resources/plugins/a+/plugin.xml

```diff
@@ -0,0 +1,19 @@
+<idea-plugin>
+  <version>0.1.101</version>
+  <idea-version since-build="193.5662" until-build="193.*"/>
+  <id>fi.aalto.cs.intellij-plugin</id>
+  <name>A+ Courses</name>
```

Comment:
Perhaps change the content (and name?) of this file to indicate it is for the testing purposes only. I can see myself in the future confused on where the change notes I just wrote disappeared when I wasn't careful enough about which file I was modifying. :P